### ISSUE TITLE
Implement currency system

### ISSUE THREAD

**Issue by rjtngit** [wetfish/online#10]
<a href="https://github.com/rempelj"><img src="https://avatars.githubusercontent.com/u/1885250?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [rempelj](https://github.com/rempelj)**
_Thursday Dec 01, 2016 at 04:10 GMT_
_Originally opened as https://github.com/rempelj/ffc/issues/11_

----

**Comment by itsrachelfish** [comment]
@rempelj How do you feel about having different tiers of currency like how WoW has bronze / silver / gold?

I think it'd be cool if there were multiple currencies like Sand Dollars / Coral / FishBux or something

**Comment by rjtngit** [comment]
Do you mean as a display format? In WoW, 100 bronze = 1 silver and 100 silver = 1 gold, right?

I'm open to it if it makes sense. I don't think converting Sand Dollars to Coral is a very easy concept to grasp from a user perspective. 

I like the name Coral as the name for the basic currency, though. We could do something like bronze coral, silver coral, etc. In the short term however, I will keep it simple with just plain ol' Coral.

FishBux can be converted to and from real life currency, correct? I don't think we should allow users to convert the basic currency (coral) to FishBux. It would be easily exploited and would probably diminish the value of FishBux to the point where it's worthless. Also, isn't there some legal stuff surrounding that? 

tl;dr I'm going to keep it simple for now.

**Comment by itsrachelfish** [comment]
sounds good! :+1:
